LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan/DevAddr.cs

```diff
@@ -25,6 +25,8 @@ namespace LoRaWan
 
         public DevAddr(uint value) => this.value = value;
 
+        public byte[] AsByteArray() => BitConverter.GetBytes(this.value);
```

Comment:
Use `BinaryPrimitives.WriteUInt32LittleEndian` instead like in the rest of the primitives and it's also explicitly clear about the endianness.